cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutorState.java

```diff
@@ -209,12 +218,40 @@ public int numFinishedLeadershipMovements() {
         execState.put("abortingPartitions", _abortingPartitionMovements);
         execState.put("abortedPartitions", _abortedPartitionMovements);
         execState.put("deadPartitions", _deadPartitionMovements);
+        if (verbose) {
+          List<Object> inProgressPartitionMovementList = new ArrayList<>();
```

Comment:
Do you think we can add the initialCapcity for
`inProgressPartitionMovementList`
`abortingPartitionMovementList`
`abortedPartitionMovementList`
`deadPartitionMovementList`
`pendingPartitionMovementList`?